[2024-03-12T20:00:29.890-0400] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: train_data.split_data scheduled__2024-03-12T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2024-03-13T00:00:29.890123+00:00 and task will be retried at 2024-03-13T00:05:28.972337+00:00.
[2024-03-12T20:00:29.891-0400] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2024-03-12T21:19:41.863-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_data.split_data scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-03-12T21:19:41.871-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_data.split_data scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-03-12T21:19:41.872-0400] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-03-12T21:19:41.895-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): split_data> on 2024-03-12 00:00:00+00:00
[2024-03-12T21:19:41.905-0400] {standard_task_runner.py:60} INFO - Started process 406490 to run task
[2024-03-12T21:19:41.910-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'train_data', 'split_data', 'scheduled__2024-03-12T00:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmpswt5y5ah']
[2024-03-12T21:19:41.911-0400] {standard_task_runner.py:88} INFO - Job 87: Subtask split_data
[2024-03-12T21:19:42.028-0400] {task_command.py:423} INFO - Running <TaskInstance: train_data.split_data scheduled__2024-03-12T00:00:00+00:00 [running]> on host DESKTOP-6U7QAQN.
[2024-03-12T21:19:42.119-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='youremail@email.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_data' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-12T00:00:00+00:00'
[2024-03-12T21:19:42.121-0400] {logging_mixin.py:188} INFO - /home/ironbong_jr/mlops/project/Group-5-Default-of-Credit-Card-Clients-IE7374/airflow/dags/dataSplit/../data/default of credit card clients.xls
[2024-03-12T21:20:15.501-0400] {python.py:202} INFO - Done. Returned value was: None
[2024-03-12T21:20:15.511-0400] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=train_data, task_id=split_data, execution_date=20240312T000000, start_date=20240313T011941, end_date=20240313T012015
[2024-03-12T21:20:15.578-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-12T21:20:15.614-0400] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
